code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -907,6 +960,25 @@ components:
       pattern: ^[A-Za-z][A-Za-z0-9_]{7,63}$
       description: Human readable name of the Application Provider.
 
+    ClusterInfo:
```

Comment:
Although the ClusterInfo object returns the additional metadata about the clusterRef but shouldn't there be a bit more about the cluster itself like number of nodes, k8s version, GPU enabled etc?